js/src/modals/CreateWallet/createWalletStore.js

```diff
@@ -283,7 +283,8 @@ export default class CreateWalletStore {
 
     const owners = _wallet.owners.filter((owner) => !/^(0x)?0*$/.test(owner));
 
-    if (_wallet.required > owners.length) {
+    // Real number of owners is owners + creator
```

Comment:
is this true for the latest wallet code?

Reply:
Okay so it seems that it hasn't been updated to the latest wallet (after the last PR was merged in `paritytech/contracts`), so `msg.sender` is still one of the owner.
We _should_ update it to the latest version. Although it would be easier with this PR merged : https://github.com/paritytech/contracts/pull/74